Features/cap05

candy184 opened the pull request.

candy184 committed:
Create README.md

Subiendo el resumen del capitulo 5 del curso de html5

candy184 committed:
Create index.html

Subiendo un ejemplo del capitulo 5 del curso de html5

semanticmx commented on cap-05/README.md:
```diff
@@ -0,0 +1,70 @@
+Introducción al <canvas>
```
Dale un poco de formato, estos pueden ser títulos

https://www.markdownguide.org/cheat-sheet/

semanticmx commented on cap-05/README.md:
```diff
@@ -0,0 +1,70 @@
+
+Ejemplo de código:
+
+javascript
```
El código ponlo con formato también

candy184 replied:
listo

semanticmx commented on cap-05/README.md:
```diff
@@ -0,0 +1,70 @@
+Gradientes
+En lugar de usar colores sólidos, puedes llenar figuras con gradientes. Los gradientes son transiciones suaves entre colores y pueden ser lineales o radiales.
+
+Ejemplo de gradiente lineal:
```
Igual para títulos y subtítulos

semanticmx commented on cap-05/src/index.html:
```diff
@@ -0,0 +1,43 @@
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Cuando tengas más de un atributo agregalos uno por línea.

así:

```html
<meta 
  name="viewport" 
  content="width=device-width, initial-scale=1.0"
>
```

candy184 replied:
listo

semanticmx replied:
creo que este cambio no se guardó, sigo viendo una sola línea `<meta name="viewport" content="width=device-width, initial-scale=1.0">`

candy184 replied:
corregido

semanticmx commented on cap-05/src/index.html:
```diff
@@ -0,0 +1,43 @@
+</head>
+<body>
+
+<h1>Canvas: Dibujar Formas Simples</h1>
```
los elementos dentro del `body` deben de ir identados

candy184 committed:
Update README.md

Corrigiendo el resumen del capitulo 5 agregándole formato al titulo de introducción

candy184 committed:
Update README.md

Agregando formato a los ejemplos y a los subtitulos de el resumen del capitulo 5

candy184 committed:
Update index.html

corrigiendo los atributos del código de HTML5 en el meta

candy184 committed:
Update index.html

Agregando la identacion al body

candy184 reviewed (commented):
@semanticmx  cambios listos

candy184 commented:
@semanticmx  cambios realizados

semanticmx commented on cap-05/README.md:
```diff
@@ -0,0 +1,93 @@
+
+javascript
+
+`for (var x = 0.5; x < 500; x += 10) {`
```
un ajuste más por favor.

lo que estás haciendo es esto:

```

javascript

`for (var x = 0.5; x < 500; x += 10) {`

`context.moveTo(x, 0);`
```

pero el texto `javascript` así suelto no hace nada

y cuando tengas bloques de código, es decir, más de una línea tienes que usar los tres acentos invertidos, es decir te tiene que quedar así:

 
```
```javascript
for (var x = 0.5; x < 500; x += 10) {
     context.moveTo(x, 0);
```
```

candy184 replied:
corregido

semanticmx commented on cap-05/README.md:
```diff
@@ -0,0 +1,93 @@
+
+### Ejemplo de cómo dibujar una imagen: ###
+
+javascript
```
aplícale el ajuste de los tres acentos invertidos a todo el código.

candy184 replied:
listo

candy184 committed:
Update README.md

corrigiendo el capitulo 5 del curso de html5

candy184 committed:
Update index.html

corrigiendo la estructura del ejemplo index.html del capitulo 5

candy184 commented:
@semanticmx  cambios listos

semanticmx reviewed (approved):
excelente!
